src/main/resources/application.properties

```diff
@@ -11,3 +11,6 @@ server.port=9003
 
 # Enables logging of each request
 logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=DEBUG
+
+# Set to true if connecting via SSL, for instance when using DHS
+mlUseSSL=false
```

Comment:
I recommend using mlSimpleSsl to match the ml-gradle property, as it's possible to have the properties in application.properties be read in by build.gradle and applied as regular Gradle properties.

Reply:
I am in favor of aligning with ml-gradle, just wondering what 'Simple' in 'mlSimpleSsl' means. Could and should there also be an 'mlAdvancedSsl'? Typical SSL requirements are not really simple (bi-directional certs, no old TLS, full-chain verification, etc etc)..

Reply:
In favour of renaming. Also, support @grtjn view here - what I did is merely to support connecting to DHS which seems to have pretty low bar when it comes to configuring SSL. I'd even go as far as renaming this to mlConnectToDhs or similar. WDYT?

Reply:
Agreed to name change.